src/DestinyService.java

```diff
@@ -6,4 +8,6 @@
 
     long getMembershipId(int membershipTypeId, String username) throws IOException;
 
+    String getCharacterSummary (int membershipTypeId, long membershipId) throws IOException;
+
```

Comment:
Delete the whitespace near between method name and brackets. Also delete the blank line below method declaration. `CTRL` + `ALT` + `L` will autoformat your code in IntelliJ.